antora-playbook.yml

```diff
@@ -1,11 +1,21 @@
 # $(npm bin)/antora --clean --fetch antora-playbook.yml
+# or
+# $(npm bin)/antora --clean --generator=@antora/site-generator-ms --attribute=export-site-navigation-data --fetch antora-playbook.yml
 site:
   title: Couchbase Docs
   url: https://docs.couchbase.com
   start_page: home::index.adoc
   robots: allow
   keys:
     google_analytics: GTM-MVPNN2
+    nav_groups: |
+      [
+        { "title": "Server", "startPage": "home::server.adoc", "components": ["server", "elasticsearch-connector", "kafka-connector", "spark-connector"] },
+        { "title": "Mobile", "startPage": "home::mobile.adoc", "components": ["couchbase-lite", "sync-gateway"] },
+        { "title": "Cloud", "startPage": "home::cloud.adoc", "components": ["cloud", "cloud-native-database", "operator", "service-broker"] },
```

Comment:
Separate CB Cloud and Cloud-Native:
{ "title": "Cloud", "startPage": "home::cloud.adoc", "components": ["cloud"] },
{ "title": "Cloud-Native", "startPage": "home::cloud-native.adoc", "components": ["cloud-native-database", "operator", "service-broker"] },

Reply:
We can't separate them until home::cloud-native.adoc exists. Otherwise, it won't get included in the navbar anyway (and the components won't be accessible).